modules/administration-guide/pages/installing-che-on-minikube-keycloak-oidc.adoc

```diff
@@ -341,3 +341,9 @@ $ {prod-cli} dashboard:open
 ----
 
 . Log in to the {prod-short} instance with *Username:* `test` and *Password:* `test`.
+
+ifeval::["{prod-id-short}" == "che"]
```

Comment:
IMO, you don't need this statement here because this page is upstream-only and does not appear in the downstream docs at all:
```suggestion
```